Verifying sv/ldv-linux-3.16-rc1/43_2a_consumption_linux-3.16-rc1.tar.xz-43_2a-drivers--usb--serial--sierra.ko-entry_point_true-unreach-call.cil.out.c
main using GenerateFunctionCallPreprocessor ...
GenerateFunctionCallPreprocessor parents for __VERIFIER_error:
__VERIFIER_error
  ldv_error
    ldv_check_alloc_nonatomic
    <end>
    ldv_check_alloc_flags_and_return_some_page
    <end>
    ldv_check_alloc_flags
      kmalloc
        sierra_setup_urb
          sierra_open
            main
            <end>
            main
            <end>
        sierra_write
          main
          <end>
          main
          <end>
      kzalloc
        sierra_startup
          main
          <end>
        sierra_port_probe
          main
          <end>
          main
          <end>
          main
          <end>
          main
          <end>
          main
          <end>
      ldv_usb_submit_urb_26
        sierra_submit_delayed_urbs
          sierra_resume
      ldv_usb_alloc_urb_25
        sierra_setup_urb
          sierra_open
      ldv_usb_submit_urb_24
        sierra_submit_rx_urbs
          sierra_resume
          sierra_open
      ldv_usb_submit_urb_23
        sierra_submit_rx_urbs
          sierra_resume
          sierra_open
      ldv_usb_submit_urb_22
        sierra_instat_callback
      ldv_usb_submit_urb_21
        sierra_indat_callback
          sierra_open
      ldv_usb_submit_urb_20
        sierra_write
      ldv_usb_alloc_urb_19
        sierra_write
      ldv_kmem_cache_alloc_16
      <end>
    ldv__builtin_trap
    <end>
